core/src/main/java/org/frankframework/core/PipeLineSession.java

```diff
@@ -55,6 +55,7 @@ public class PipeLineSession extends HashMap<String,Object> implements AutoClose
 	public static final String MESSAGE_ID_KEY       = "mid";        // externally determined (or generated) messageId, e.g. JmsMessageID, HTTP header configured as messageId
 	public static final String CORRELATION_ID_KEY   = "cid";       // conversationId, e.g. JmsCorrelationID.
 	public static final String STORAGE_ID_KEY       = "key";
+	public static final String MANUAL_RETRY_KEY     = "_isManualRetry";
```

Comment:
Moet deze niet dubbel underscore geprefixed zijn: `PipeLineSession.SYSTEM_MANAGED_RESOURCE_PREFIX`.
Hij is niet closeable maar dan kunnen we hem wel 'afschermen' voor een gebruiker.